### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by stephencelis**
Gonna accumulate some small things here.

### CODE REVIEW

File: Sources/PointFree/Account/UpdateProfile.swift [hunk 1]
```diff
@@ -27,17 +27,23 @@ extension ProfileData: Decodable {
...
+}
+
 let updateProfileMiddleware =
-  filterMap(require1 >>> pure, or: redirect(to: .account(.index)))
-    <<< filterMap(require2 >>> pure, or: loginAndRedirect)
+  filterMap(require2 >>> pure, or: loginAndRedirect)
+    <<< filterMap(require1 >>> pure, or: redirect(to: .account(.index)))
+    <<< filter(
+      get1 >>> ^\.email >>> isValidEmail,
+      or: redirect(to: .account(.index), headersMiddleware: flash(.error, "Please enter a valid email."))
```
Comment (mbrandonw) [inline Sources/PointFree/Account/UpdateProfile.swift]: nice nice

File: Sources/PointFreeTestSupport/PointFreeTestSupport.swift [hunk 2]
```diff
@@ -217,27 +217,15 @@ extension Stripe.Subscription.Item {
...
-    else { return request }
-
-  request.allHTTPHeaderFields = (request.allHTTPHeaderFields ?? [:])
-    .merging(["Cookie": "pf_session=\(sessionCookie)"], uniquingKeysWith: { $1 })
-
-  return request
-}
-
-public func unauthedRequest(to route: Route) -> URLRequest {
+public func request(to route: Route, session: Session = .loggedOut) -> URLRequest {
```
Comment (mbrandonw) [inline Sources/PointFreeTestSupport/PointFreeTestSupport.swift]: much better

File: Tests/LinuxMain.swift [hunk 3]
```diff
@@ -68,6 +68,21 @@ extension HtmlCssInlinerTests {
     ("testHtmlCssInliner", testHtmlCssInliner)
   ]
 }
+extension InviteTests {
```
Comment (mbrandonw) [inline Tests/LinuxMain.swift]: dang, wonder if we should have a test phase that runs `make sourcery` and sees if `git status` outputs anything

Comment (stephencelis) [reply Tests/LinuxMain.swift]: I wonder if it's possible to hook into build phases while running `swift generate-xcodeproj`.

**Pull request merged**